lib/invoice_repo.rb

```diff
@@ -29,6 +29,12 @@ def find_all_by_customer_id(id)
    end
  end
 
+ def find_all_successful_transactions_by_merchant_id(id)
```

Comment:
Keep this code on your local, we may need it after we track enumerables

Reply:
![image](https://user-images.githubusercontent.com/32661560/34792602-ceb74630-f605-11e7-8973-589d657a5aea.png)

Reply:
Cool, I'm gonna start tracking stuff out now and we'll just leave that there.